core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -106,7 +106,7 @@ public static boolean waitCompleted(final FileSystem fs, final AlluxioURI uri,
         ClientContext.getConf().getLong(Constants.USER_FILE_WAITCOMPLETED_POLL_MS);
     boolean completed = false;
     long timeleft = deadline - System.currentTimeMillis();
-    long toSleep = 0;
+    long toSleep;
```

Comment:
we should move this as close to its usage as possible, so just after `} else if (!completed) {`